src/wp-includes/theme.php

```diff
@@ -514,7 +514,7 @@ function search_theme_directories( $force = false ) {
 		// Start with directories in the root of the active theme directory.
 		$dirs = @ scandir( $theme_root );
 		if ( ! $dirs ) {
-			trigger_error( "$theme_root is not readable", E_USER_NOTICE );
+			wp_trigger_error( __FUNCTION__, "$theme_root is not readable", E_USER_NOTICE );
```

Comment:
The `E_USER_NOTICE` is the default. Can safely remove it.

```suggestion
			wp_trigger_error( __FUNCTION__, "$theme_root is not readable" );
```